test/functional/feature_governance.py

```diff
@@ -136,6 +136,16 @@ def run_test(self):
         trigger_data = list(valid_triggers.values())[0]
         assert_equal(trigger_data['YesCount'], 1)
 
+        # Make sure amounts aren't trimmed
+        payment_amounts_expected = [str(satoshi_round(str(p0_amount))), str(satoshi_round(str(p1_amount))), str(satoshi_round(str(p2_amount)))]
+        data_string = list(self.nodes[0].gobject("list", "valid", "triggers").values())[0]["DataString"]
+        payment_amounts_trigger = json.loads(data_string)["payment_amounts"].split("|")
+        for amount_str in payment_amounts_trigger:
+            if amount_str in payment_amounts_expected:
+                pass
+            else:
+                assert False
```

Comment:
nit: it can be simplified. `assert False` doesn't give any extra information anyway:

```suggestion
            assert(amount_str in payment_amounts_expected)
```